pootle/core/search/base.py

```diff
@@ -16,6 +16,8 @@
 class SearchBackend(object):
     def __init__(self, config_name=None, **kwargs):
         self._setup_settings(config_name)
+        self.weight = 1
```

Comment:
Put it in the config decription. Boring people with extra info is not helpful when they don't need to config it if they don't need or understand it.